src/components/ProductDetail/ProductDetail.jsx

```diff
@@ -53,12 +103,19 @@ export function ProductDetail() {
         <Title>{name}</Title>
         <Category>{slug.replaceAll('--', ' & ')}</Category>
         <Description>{description[0].text}</Description>
-        <AmountWrapper>
-          <Price>$ {price}</Price>
-          <Stock>{stock} in stock</Stock>
-          <Input max={stock}/>
-          <AddToCartButton>ADD TO CART</AddToCartButton>
-        </AmountWrapper>
+        {actualStock < 1 
+          ? <OutOfStock>OUT OF STOCK</OutOfStock>
+          : (<AmountWrapper>
+              <Price>$ {price}</Price>
+              <Stock>{actualStock} in stock</Stock>
+              <Input
+                max={actualStock}
+                onChange={handleChange}
```

Comment:
I see the `max` parameter is correctly configured, although we're missing one more validation over the `handleClick` function so we avoid going over the `actualStock` when directly typing
![GIF Recording 2022-07-12 at 4 22 50 p m](https://user-images.githubusercontent.com/40035747/178605327-db51f2d2-0b51-4c3e-8f08-8ac9170ee462.gif)
,